gli4py/glinet.py

```diff
@@ -171,6 +170,69 @@ async def connected_clients(self) -> dict:
                 clients[client['mac']] = client
         return clients
 
+    async def _wifi_config_get(self) -> dict:
+        return await self._request(self.gen_sid_payload('call', ['wifi', 'get_config'], self.sid))
+
+    async def _wifi_config_set(self, config: dict) -> dict:
+        return await self._request(self.gen_sid_payload('call', ['wifi', 'set_config', config], self.sid))
+    
+    async def wifi_ifaces_get(self) -> dict:

```

Comment:
We can improve typing here
```suggestion
    async def wifi_ifaces_get(self) -> dict[str,dict[str,Any]]:
```